doc/user/content/guides/postgres-cloud.md

```diff
@@ -123,6 +121,44 @@ As a user with the `cloudsqlsuperuser` role, make these changes to the upstream
 
 For more information, see the [Cloud SQL](https://cloud.google.com/sql/docs/postgres/replication/configure-logical-replication#configuring-your-postgresql-instance) documentation.
 
+## Azure Database for PostgreSQL
+
+Before you start, note that a database restart will be required after making changes to the database.
+
+1. The Materialize replica will need access to connect to the upstream database. In your Azure portal, go to the Azure Database for PostgreSQL instance and under the "Connections security" section add your [Materialize instance's IP address](/cloud/security/#static-ip-addresses) to the allowed IP addresses list and click on the "Save" button.
+
+1. Enable Logical Replication by going to the Azure Database for PostgreSQL instance, then under the "Replication" section click the "Enable Logical Replication" toggle button and click on "Save".
```

Comment:
This is specific to setting the configuration through the Azure portal. The instructions should point to the parameter that needs to be toggled and its value, independent of [how](https://docs.microsoft.com/en-us/azure/postgresql/concepts-logical#set-up-your-server) you choose to set it.